build-system/tasks/extension-helpers.js

```diff
@@ -673,11 +660,16 @@ function buildNpmBinaries(extDir, options) {
           'preact/dom': 'react-dom',
         },
         wrapper: '',
-      });
-    }
-    return buildBinaries(extDir, binaries, options);
-  });
-  return Promise.all(promises);
+      },
+      bento: {
```

Comment:
Note: in the future we'll need to add externals for ['preact', 'preact/dom', 'preact/compat', 'preact/hooks']. But that'll need to go in tandem with @alanorozco  work to create a `bento.js` "runtime" for the bento components